packages/ui/Routable/tests/Router-specs.js

```diff
@@ -36,133 +37,108 @@ describe('Router', () => {
 	};
 
 	test('should render a single component matching the {path}', () => {
-		const subject = shallow(
-			<RouterBase routes={routes} path="/app" />
-		);
+		render(<RouterBase routes={routes} path="/app" />);
+		const view = screen.getByTestId('button');
```

Comment:
Let's use `getByRole('button')` instead of using `data-testid`.
Actually, React Testing Library doesn't recommend using `data-testid` where you can query by other attributes.
Please take a look at https://testing-library.com/docs/queries/bytestid.